server/services/editReports/useOfForceDetails.ts

```diff
@@ -0,0 +1,197 @@
+import R from 'ramda'
+import { QUESTION_SET } from '../../config/edit/useOfForceDetailsConfig'
+
```

Comment:
This determines the edits made. It compares the new body data with the corresponding section within the database. It returns the edit data, the original persisted data and a 'hasChanged' boolean that identifies whether the two are different